client/model-config.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
re: extended json format

Are we still getting benefit from using the JSON+loopback-boot combination? Could we replace the static JSON with equivalent JS instead? We already have to define the datasource for the client using the API instead of JSON anyway..